### PR TITLE
Check graph is multigraph when `InitEdge` or `AddEdgeLabel`

### PR DESCRIPTION
**Pull request by acezen**
What do these changes do?
-------------------------

Related issue number
--------------------

N/A

### CODE REVIEW

File: modules/graph/fragment/arrow_fragment.h [hunk 1]
```diff
@@ -1878,6 +1881,8 @@ class ArrowFragment
       }
     }
 
+    LOG(INFO) << "existed_parallel_edge=" << existed_parallel_edge;
```
Comment (sighingnow) [inline modules/graph/fragment/arrow_fragment.h:1884]: `VLOG(2)` would be better here.

**Comment by sighingnow** [comment]
Please rebase and sign-off your commits as well.

File: modules/graph/fragment/arrow_fragment.h (hunk 1 above)
Comment (acezen) [reply modules/graph/fragment/arrow_fragment.h]: removed

**Comment by acezen** [comment]
cc/ @siyuan0322

**Pull request merged**